README.md

```diff
@@ -3,6 +3,9 @@
 		<img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
 	</a>
 </div>
+
+<h1 align="center">Webpack CLI</h1>
```

Comment:
```suggestion
<h1 align="center">webpack CLI</h1>
```
Please make it lowercase as per [branding guidlines](https://webpack.js.org/branding/#the-name)